docker/Dockerfile-12

```diff
@@ -0,0 +1,95 @@
+# Base Information
```

Comment:
Seems like only difference is version number. How about adding build argument rather than new Dockerfile?
ex.
```Dockerfile
ARG LLVM_VERSION=10 # default version
# ...
# Install LLVM
RUN wget https://apt.llvm.org/llvm.sh  \
    && chmod +x ./llvm.sh \
    && ./llvm.sh {LLVM_VERSION} \
# ...
```
```bash
# Set LLVM version with build argument
docker buildx build -f docker/Dockerfile -t utopia --build-arg LLVM_VERSION=12 .
```